fhir-server-test/src/test/resources/nats/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+networks:
+  main:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.19.0.0/24
+      driver: default
+services:
+    nats-node1:
+        image: nats-streaming:0.17.0
+        container_name: nats-node1
+        volumes:
+            - .:/nats
+        command: --config /nats/config-1.conf
+        networks:
+            - main
+        ports:
+            - 4222:4222
+            - 6222:6222
```

Comment:
```suggestion
            - 4222:4222
```

Reply:
seems like 6222 is the "backend" port and so no need to expose it on the host?  please correct me if I'm wrong

Reply:
6222 is an HTTP port, but we don't use it for this example, so it can be excluded.